src/NHibernate.Test/LazyProperty/LazyPropertyFixture.cs

```diff
@@ -50,7 +50,7 @@ protected override void OnTearDown()
 			using (var s = OpenSession())
 			using (var tx = s.BeginTransaction())
 			{
-				Assert.That(s.CreateSQLQuery("delete from Book").ExecuteUpdate(), Is.EqualTo(1));
+				Assert.That(s.CreateSQLQuery("delete from Book").ExecuteUpdate(), Is.EqualTo(2));
```

Comment:
Since I have commented on the commit (no PR at that time), the PR was not showing the line I was commenting. This was this one.

I consider it should be changed to: 
```c#
s.CreateQuery("delete from Book").ExecuteUpdate();
```

There are no reason to assert the number of inserted things in the teardown. So better remove this undue old assert rather than fixing it. (By the way, it is better to use a HQL delete rather than a SQL one, so I have changed that too in the above code.)